MovieQuiz/Presentation/ResultAlertPresenter.swift

```diff
@@ -0,0 +1,28 @@
+// swiftlint: disable all
+
+import Foundation
+import UIKit
+
+struct ResultAlertPresenter {
+    var viewController: UIViewController
```

Comment:
К сожалению в уроке не было информации про retain cycle (вы можете погуглить самостоятельно), урок обещали обновить в будущем. Однако сейчас предлагаю ослабить ссылку через weak 
```suggestion
    weak var viewController: UIViewController?
```

Текущее решение приводит к утечкам памяти